app/styles/partials/_info.scss

```diff
@@ -248,4 +248,58 @@ $packages: (
     width: 30px;
     order: 1;
   }
-}
\ No newline at end of file
+}
+
+.contributors img {
+  border-radius: 50%;
+  border: 2px solid #fff;
+  width: 30px;
+  margin-right: 5px;
+}
+
+/* Tooltip container */
+.tooltip {
+  position: relative;
+  display: inline-block;
+}
+
+/* Tooltip text */
+.tooltip .tooltiptext {
```

Comment:
I would suggest using sass potential here and use nested selectors, instead of writing multiple classes selector typical for css syntax

Reply:
Good point! Fixed